package.json

```diff
@@ -34,33 +34,33 @@
-		"docker:env":"echo \"Docker package scripts are deprecated. Please use `jetpack docker <cmd>` instead\"",
-		"docker:install":"echo \"Docker package scripts are deprecated. Please use `jetpack docker <cmd>` instead\"",
-		"docker:jt-config":"echo \"Docker package scripts are deprecated. Please use `jetpack docker <cmd>` instead\"",
-		"docker:jt-down":"echo \"Docker package scripts are deprecated. Please use `jetpack docker <cmd>` instead\"",
-		"docker:jt-up":"echo \"Docker package scripts are deprecated. Please use `jetpack docker <cmd>` instead\"",
-		"docker:multisite-convert":"echo \"Docker package scripts are deprecated. Please use `jetpack docker <cmd>` instead\"",
+		"docker:build": "printf '\\e[30;43m`pnpm run docker:build` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker build",
+		"docker:build-image": "printf '\\e[30;43m`pnpm run docker:build-image` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker build-image",
+		"docker:clean": "printf '\\e[30;43m`pnpm run docker:clean` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker clean",
+		"docker:compose": "printf '\\e[30;43m`pnpm run docker:compose` is deprected. Now it is integrated into relevant Jetpack CLI commands.\\e[0m\\n';",
+		"docker:compose-extras": "printf '\\e[30;43m`pnpm run docker:compose-extras` is deprected. Now it is integrated into relevant Jetpack CLI commands\\e[0m\\n';",
+		"docker:compose-volumes": "printf '\\e[30;43m`pnpm run docker:compose-volumes` is deprected. Now it is integrated into relevant Jetpack CLI commands\\e[0m\\n';",
+		"docker:db": "printf '\\e[30;43m`pnpm run docker:db` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker db",
+		"docker:down": "printf '\\e[30;43m`pnpm run docker:down` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker down",
+		"docker:env": "printf '\\e[30;43m`pnpm run docker:env` is deprected. Now it is integrated into relevant Jetpack CLI commands.\\e[0m\\n';",
+		"docker:install": "printf '\\e[30;43m`pnpm run docker:install` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker install",
+		"docker:jt-config": "printf '\\e[30;43m`pnpm run docker:jt-config` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker jt-config",
+		"docker:jt-down": "printf '\\e[30;43m`pnpm run docker:jt-down` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker jt-down",
+		"docker:jt-up": "printf '\\e[30;43m`pnpm run docker:jt-up` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker jt-up",
+		"docker:multisite-convert": "printf '\\e[30;43m`pnpm run docker:multisite-convert` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker multisite-convert",
 		"docker:ngrok": "pnpm run docker:compose -- -f tools/docker/docker-compose-ngrok.yml",
 		"docker:ngrok-down": "pnpm run docker:ngrok -- down",
 		"docker:ngrok-up": "pnpm run docker:ngrok -- up",
```

Comment:
These are all broken now, as you made `docker:compose` a no-op. If nothing else, you should have these output an appropriate message stating that ngrok isn't supported anymore instead of implying that there's a `jetpack docker` subcommand that'll work.

Reply:
I haven't touched these, since I'm not sure if it worth supporting Ngrok further. I started the internal convo: p9dueE-2Vm-p2